src/Sentry/SentryOptions.cs

```diff
@@ -360,6 +361,12 @@ public bool Debug
         /// </remarks>
         public DeduplicateMode DeduplicateMode { get; set; } = DeduplicateMode.All ^ DeduplicateMode.InnerException;
 
+        /// <summary>
+        /// Path to the directory where events are stored locally for resilience purposes.
+        /// If set to <code>null</code>, caching will not be used.
+        /// </summary>
+        public DirectoryInfo? CacheDirectory { get; set; }
```

Comment:
I think `string` is probably a better type, to for example bind from configuration files

Reply:
Hm, I didn't think about configuration files. That may indeed be a good reason to change it to `string`. Ah, unfortunate that C# discourages proper static typing so much.